proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/sdk/PowerAuthSDK.java

```diff
@@ -2530,22 +2545,16 @@ public void onBiometricDialogFailed(@NonNull PowerAuthErrorException error) {
     @Nullable
     public ICancelable signJwtWithDevicePrivateKey(@NonNull Context context, @NonNull PowerAuthAuthentication authentication, @NonNull Map<String, Object> claims, @NonNull IJwtSignatureListener listener) {
         final JsonSerialization serialization = new JsonSerialization();
-        final byte[] serializedClaims = serialization.serializeObject(claims);
-        return signDataWithDevicePrivateKey(context, authentication, serializedClaims, new IDataSignatureListener() {
+        final String jwtHeader = "eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9"; // {"alg":"ES256","typ":"JWT"}
```

Comment:
This feels a bit arbitrary, could we extract it to some constant, i.e., `JwtHeader.ES256`?

Reply:
Objective-C is already enough ugly language, so adding 4-5 more lines just to be "nice" makes no sense to me :)